src/local_scheduler/local_scheduler.cc

```diff
@@ -155,19 +155,6 @@ void LocalSchedulerState_free(LocalSchedulerState *state) {
    * the possibility of orphan worker processes. */
```

Comment:
What's the rationale for switching around the order of the cleanup in `LocalSchedulerState_free`?